src/data/pokemon/species_info/gen_1.h

```diff
@@ -10220,7 +10220,7 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
         .types = { TYPE_FIGHTING, TYPE_FIGHTING },
         .evYield_Attack = 2,
         .abilities = { ABILITY_INTIMIDATE, ABILITY_ANGER_POINT, ABILITY_CUD_CHEW },
-        .bodyColor = BODY_COLOR_BROWN,
+        .bodyColor = BODY_COLOR_BLACK,
```

Comment:
I'd agree that this is probably out of scope (just lmk if you don't want to do it), but all the traits shared between Paldean Tauros forms should probably be inside the base stats macro (which should be renamed appropriately in that case)

Reply:
it was handled in #3971, I'll merge with upcoming.